bin/q

```diff
@@ -611,7 +617,7 @@ class TableColumnInferer(object):
                     # in relaxed mode, add columns to fill the missing ones
                     self.header_row = self.header_row + \
                         ['c%s' % (x + len(self.header_row) + 1)
-                         for x in xrange(self.column_count - len(self.header_row))]
+                         for x in range(self.column_count - len(self.header_row))]
```

Comment:
If __x__ is really large __range(x)__ on Python 2 has memory overhead but if x is small who cares.  In general I use __from six import xrange__ instead.

Reply:
👍 decided not to do it for now, like you said in most cases this should be a non issue. I'll do some benchmarks later on about the difference between py2 and py3 behavior, and fix this if needed.